src/BloomExe/Edit/WebThumbNailList.cs

```diff
@@ -707,6 +707,11 @@ private void GridReordered(string s)
 						"That change is not allowed. Front matter and back matter pages must remain where they are");
 					var caption = LocalizationManager.GetString("PageList.CantMoveXMatterCaption",
 						"Invalid Move");
+					if (_pages[i].Book.LockedDown)
+					{
+						msg = LocalizationManager.GetString("PageList.CantMoveVernacular",
+							"Pages can't be re-ordered when you are translating a book");
```

Comment:
Contractions are to be avoided in easy english (http://www.easyenglish.info/team/eestyle.htm).